var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -61,19 +61,30 @@ def autoreconf(self, spec, prefix):
         aclocal('-I', 'lib/autoconf')
         autoconf()
 
+    @property
+    def optflags(self):
+        flags = '-O2'
+
+        # Optimizations for the Intel compiler, suggested by CP2K
+        if '%intel' in self.spec:
+            flags += ' -xAVX -axCORE-AVX2 -ipo'
```

Comment:
@adamjstewart I'll go with @lee218llnl suggestion, as it will probably result in a good enough performance on most machines without the need to tweak `compilers.yaml`.